packages/teleterm/src/ui/DocumentTerminal/useDocumentTerminal.ts

```diff
@@ -81,7 +81,7 @@ async function initState(
 
   const clusterUri = routing.getClusterUri(doc);
   const rootCluster = ctx.clustersService.findRootClusterByResource(clusterUri);
-  const cmd = createCmd(doc, rootCluster.proxyHost, getClusterActualName());
+  const cmd = createCmd(doc, rootCluster.proxyHost, getClusterName());
```

Comment:
Some change somewhere broke connecting to SSH servers on leaf clusters. When I'm trying to connect, I get this error in the terminal tab:

> ERROR: failed connecting to node 4bbde2c9-b805-4dd8-8032-a0bed162d85b. unable to locate node matching UUID-like target 4bbde2c9-b805-4dd8-8032-a0bed162d85b

(Attaching a comment to this line of code just because it's related, I didn't actually check if it's the one causing problems).

Reply:
Ah, I was wrong when I removed `leafClusterId` from `DocumentTshNode`. It is should be passed to `routing.getClusterUri(doc)` (from the comment above) to get server from the correct cluster. 
I reverted that commit, because `Terminal` component should also work for kube documents, so `leafClusterId` is needed as well.

Reading `leafClusterId` and `rootClusterId` explicitly helped find this bug 😏

Reply:
I'm not sure why TypeScript can't infer that those fields are read there, after all TypeScript is okay with passing a doc there as `Params`.